### PR TITLE
Descriptions update

### PR DESCRIPTION
**Pull request by Jannes123**
@timlinux  please review

This is for #1836 scrollbars

and #1785 Hardcoded options

### CODE REVIEW

File: safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py [hunk 1]
```diff
@@ -107,6 +108,17 @@ def __init__(self):
 
         # AG: Use the proper minimum needs, update the parameters
         self.parameters = add_needs_parameters(self.parameters)
+        self.hardcode_parameters = OrderedDict([
```
Comment (timlinux) [inline safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py]: can we call this 'itb_parameters' rather?

File: safe/defaults.py [hunk 2]
```diff
@@ -140,6 +140,7 @@ def default_gender_postprocessor():
     gender = BooleanParameter()
     gender.name = 'Gender'
     gender.value = True
+    gender._description = 'Enable a gender specific breakdown in the report'
```
Comment (timlinux) [inline safe/defaults.py]: Just note that the changes made to defaults.py are going to conflict with my work in #1792

File: safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py [hunk 3]
```diff
@@ -107,6 +108,17 @@ def __init__(self):
 
         # AG: Use the proper minimum needs, update the parameters
         self.parameters = add_needs_parameters(self.parameters)
+        self.hardcoded_parameters = OrderedDict([
+            ('x', 0.62275231), ('y', 8.03314466),  # Model coefficients
+            # Rates of people displaced for each MMI level
+            ('displacement_rate', {1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 1.0,
+                                   7: 1.0, 8: 1.0, 9: 1.0, 10: 1.0}),
```
Comment (cchristelis) [inline safe/impact_functions/earthquake/itb_earthquake_fatality_model/impact_function.py]: Pull left. #Pylintpolice.

**Comment by cchristelis** [comment]
Hi @Jannes123,

I had a look at the hard coded paramter work, LGTM. 
It looks like this PR also contains some options dialog work. I haven't looked at that in too much details, as I think it makes the most sense to actually try it out.
Two general comments: It is best to separate unrelated work into separate pull requests. Make sure that when you touch code that it ends up pylint free and conforming to our coding standard.

**Comment by timlinux** [comment]
LGTM

**Pull request merged**